test/conformance/exp_command_buffer/invalid_update.cpp

```diff
@@ -151,11 +155,11 @@ TEST_P(InvalidUpdateTest, NotUpdatableCommandBuffer) {
         nullptr,         // pNewLocalWorkSize
     };
 
-    // Update command to command-buffer that doesn't have updatable set should
-    // be an error
+    // Since no command handle was returned Update command to command-buffer
+    // should also be an error.
     ur_result_t result =
         urCommandBufferUpdateKernelLaunchExp(test_command_handle, &update_desc);
-    EXPECT_EQ(UR_RESULT_ERROR_INVALID_OPERATION, result);
+    EXPECT_EQ(UR_RESULT_ERROR_INVALID_NULL_HANDLE, result);
```

Comment:
I can't see `UR_RESULT_ERROR_INVALID_NULL_HANDLE` in the YML errors for `UpdateKernelLaunchExp`, could you add it (wonder if this means we don't need the `UR_RESULT_ERROR_INVALID_COMMAND_BUFFER_COMMAND_HANDLE_EXP`  error code, but that doesn't need to stop this PR)

Reply:
This is handled automatically by the validation layer with any handle parameter, and I believe the expected best practice is not to clutter the YAML or force runtime checks for this for that reason.